octoprint_displayprogress/templates/displayprogress_settings.jinja2

```diff
@@ -0,0 +1,26 @@
+
+<form class="form-horizontal">
+
+<div class="control-group">
+    <label class="control-label">Selection Preview</label>
+    <span id="preview" class="{{ plugin_displayprogress_barsymbol }}">
```

Comment:
This should use something like `data-bind="css: settings.plugins.displayprogress.barsymbol"` or similar instead - this SHOULD actually get rid of a requirement for extra JS altogether.